### DISCUSSION TITLE
Blackhole / Pro Tools / RME / Zoom

### DISCUSSION THREAD

**Discussion by Durdon1** [ExistentialAudio/BlackHole#131] [General]
Hi, i hope you can help.
i will transport my audio from pro tools to zomm. it is working. but i will route it to my rme fireface to and a microphone from the rme to zoom.
Can anyone help me how i got to route it?

Thanks

**Comment by Durdon1** [comment]
Hi, basicly it works now. If you will, i can post some pics. BUT the next problem is:
I will have my external mic --> RME fireface --> Zoom
Can anyone explain how?
The Input in Zoom is Blackhole 16ch cause here it takes the input frmo Pro Tools.
Any ideas?

**Comment by devinroth** [comment]
Posts some pictures please.
